### DISCUSSION TITLE
New to Starship; need help deciphering the prompt

### DISCUSSION THREAD

**Discussion by crawfordlong** [starship/starship#6351] [General]
I'm sorry for this question, lol, but what is the "[$]" part of this prompt? This is in a git repo containing nix configuration files (y'all probably figured that part out), with everything properly pushed and pulled. I haven't made any configuration changes. This is whatever results from setting starship.enabled in the relevant nix configuration file.

![screenshot-2024-10-24-16:01:40](https://github.com/user-attachments/assets/f70befcc-76be-4d22-b4d4-d23a9d3306cc)

**Comment by davidkna** [comment]
`starship explain` will give you an overview of what each module in the prompt represents. For the `git_status` module (`[$]`, `$` means something is stashed), please consult [the module documentation](https://starship.rs/config/#git-status) for the meaning of the different symbols.

**Reply by crawfordlong** [reply]
Blast. I checked that documentation and simply missed it. Thank you for the prompt (ugh sorry) reply.
